libgit/repo.go

```diff
@@ -493,7 +493,16 @@ func DeleteRepo(
 
 	_, _, err = kbfsOps.Lookup(ctx, repoNode, normalizedRepoName)
 	if err != nil {
-		return err
+		// For the common "repo doesn't exist" case, use an error type that the
+		// client can recognize.
+		switch errors.Cause(err).(type) {
+		case libkbfs.NoSuchNameError:
+			return libkb.RepoDoesntExistError{
```

Comment:
Also, it might be worth returning this for the above lookup on `kbfsRepoDir` as well, which could happen if someone tries to delete a repo in a TLF that has never had any repos at all.

Reply:
Sure, I'll add that in for consistency.